src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+// eslint-disable-next-line promise/param-names
+const promise2 = new Promise((_, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const onSuccess = () => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+};
+
+const onError = () => {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+};
+
+promise1.then(onSuccess).catch(onError);
+promise2.then(onSuccess).catch(onError);
```

Comment:
Both `promise1` and `promise2` are handled independently. This means that `onError` will be called for `promise2` after 3 seconds, regardless of whether `promise1` resolves. If the intention is to wait for either promise to resolve or reject, consider using `Promise.race` or `Promise.all` to handle them together.